src/components/NcRelatedResourcesPanel/NcTeamResources.vue

```diff
@@ -0,0 +1,293 @@
+						{{ provider.name }}
+					</h6>
+					<ul>
+						<li v-for="resource in provider.resources" :key="resource.url" class="related-team-resource">
+							<a :href="resource.url" class="related-team-resource__link">
+								<span v-if="resource.iconEmoji" class="resource__icon">
+									{{ resource.iconEmoji }}
+								</span>
+								<NcIconSvgWrapper v-else-if="resource.iconSvg"
+									class="resource__icon"
+									:svg="resource.iconSvg"
+									:size="20" />
+								<span v-if="resource.iconUrl" class="resource__icon">
+									<img :src="resource.iconURL" alt="">
+								</span>
+								<span class="resource__name">
+									{{ resource.label }}
+								</span>
+							</a>
+						</li>
+					</ul>
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import axios from '@nextcloud/axios'
+import { generateOcsUrl } from '@nextcloud/router'
+import AccountGroup from 'vue-material-design-icons/AccountGroup.vue'
+import ChevronDown from 'vue-material-design-icons/ChevronDown.vue'
+import ChevronUp from 'vue-material-design-icons/ChevronUp.vue'
+import OpenInNew from 'vue-material-design-icons/OpenInNew.vue'
+import NcButton from '../NcButton/NcButton.vue'
+import NcActions from '../NcActions/NcActions.vue'
+import NcActionLink from '../NcActionLink/NcActionLink.vue'
+import NcIconSvgWrapper from '../NcIconSvgWrapper/NcIconSvgWrapper.vue'
+
+import { t } from '../../l10n.js'
+
+export default {
+	name: 'NcTeamResources',
+
+	components: {
+		AccountGroup,
+		ChevronDown,
+		ChevronUp,
+		OpenInNew,
+		NcButton,
+		NcActions,
+		NcActionLink,
+		NcIconSvgWrapper,
+	},
+
+	props: {
+		providerId: {
+			type: String,
+			default: null,
+		},
+		itemId: {
+			type: [String, Number],
+			default: null,
+		},
+		resourceType: {
+			type: String,
+			default: '',
+		},
+		/**
+		 * Set the maximum number of resources to load
+		 */
+		limit: {
+			type: Number,
+			default: 0,
+		},
+		/**
+		 * Only used by the files sidebar
+		 *
+		 * File info is passed when registered with `OCA.Sharing.ShareTabSections.registerSection()`
+		 */
+		fileInfo: {
+			type: Object,
+			default: null,
+		},
+		/**
+		 * Make the header name dynamic
+		 */
+		header: {
+			type: String,
+			default: t('Related resources'),
+		},
+		headerOther: {
+			type: String,
+			default: t('Other resources'),
+		},
+		description: {
+			type: String,
+			default: t('Anything shared with the same group of people will show up here'),
+		},
+		/**
+		 * If this element is used on a primary element set to true for primary styling.
+		 */
+		primary: {
+			type: Boolean,
+			default: false,
+		},
+	},
+
+	data() {
+		return {
+			appEnabled: OC?.appswebroots?.circles !== undefined && (OC.config.version.split('.')[0] ?? 0) >= 29,
+			loading: false,
+			error: null,
+			teamResources: null,
+			teamOpen: null,
+		}
+	},
+
+	computed: {
+		isVisible() {
+			return !this.loading && this.error === null && this.teamResources?.length > 0
```

Comment:
Why do we need to check for `this.loading` and `this.error` here? If `this.teamResources` is set and has elements, they should be displayed, no?

Do we expect dynamic changing of `this.teamResources` (e.g. changing contexts)? In this case, hiding the resources while loading might make sense to not display outdated resources. And in this case evaluating `this.error` might make sense as well, to make sure that outdated don't get displayed. But I'd rather say we should reset `this.teamResources = null` in case of errors (line 208).